test/com/google/enterprise/adaptor/TestHelper.java

```diff
@@ -87,17 +89,6 @@ public void setAuthzAuthority(AuthzAuthority authzAuthority) {}
     return getDocIds(adaptor, Collections.<String, String>emptyMap());
   }
 
-  public static byte[] getDocContent(Adaptor adaptor, DocId docId)
```

Comment:
Thanks for the consideration.  

Agree with John that reaching into test subdir is tacit agreement to higher level of disruption.